packages/oui-navbar/navbar.less

```diff
@@ -180,6 +180,50 @@
     /* stylelint-enable indentation */
   }
 
+  &-notification {
+    display: flex;
+    padding: @oui-notification-padding;
+
+    &__icon {
+      margin: @oui-notification-icon-margin;
+    }
+
+    &__icon .oui-icon {
+      font-size: @oui-notification-icon-font-size;
+
+      &.oui-icon-info_circle {
+        color: @oui-notification-info-icon-color;
+      }
+
+      &.oui-icon-success_circle {
+        color: @oui-notification-success-icon-color;
+      }
+
+      &.oui-icon-warning_circle {
+        color: @oui-notification-warning-icon-color;
+      }
+
+      &.oui-icon-error_circle {
+        color: @oui-notification-error-icon-color;
+      }
+    }
+
+    &__date {
+      padding: 0.5rem 0;
+    }
+
+    &__description {
+      line-height: 2rem;
```

Comment:
rem-calc? for others too

Reply:
```rem-calc(...)``` is usefull to convert pixels into rem, it already rems, no conversion needed.

But, default unit for ```line-heigt``` is rem so I could do ```line-height: 2;```

Reply:
ok for line-height
but others too...
font-size: 1.6rem;
max-height: 8rem;
Search in the project, rem-calc is used everywhere except 4-5 places